spec/decorators/decidim/proposals/import/proposal_creator_decorator_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "decidim/proposals/import/proposal_creator"
+
+describe Decidim::Proposals::Import::ProposalCreator do
+  subject { described_class.new(data, context) }
+
+  let!(:moment) { Time.current }
+  let(:data) do
+    {
+      id: 1337,
+      "id" => "101",
+      category: category,
+      scope: scope,
+      :"title/en" => Faker::Lorem.sentence,
+      :"body/en" => Faker::Lorem.paragraph(sentence_count: 3),
+      address: "#{Faker::Address.street_name}, #{Faker::Address.city}",
+      latitude: Faker::Address.latitude,
+      longitude: Faker::Address.longitude,
+      component: component,
+      published_at: moment
```

Comment:
can you unify syntax? there are `key: value` and `key => value` syntax, but arrow is an obsolete way.  And why two ids?